.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+env:
+  GIT_COMMIT_SHA: ${{ github.sha }}
+  GIT_BRANCH: ${{ github.ref }}
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_WITH: lint
+      BUNDLE_WITHOUT: development:test
```

Comment:
`lint` isn't an environment, so `BUNDLE_WITH` won't do anything, no? And do we need a `BUNDLE_WITHOUT` since the only thing we're doing is the test environment?

Reply:
Ah yes, I've pushed these changes without also pushing an updated Gemfile. The idea is to have 2 groups: lint and development, test.
These lines will only install linting-related gems for the Linting job, greatly increasing installation time.

I should have removed it for now as it's not actually used yet, apologies for the confusion!